upload.php

```diff
@@ -1 +1,20 @@
 <?php
+
+if ( $_SERVER['REQUEST_METHOD']=='GET' && realpath(__FILE__) == realpath( $_SERVER['SCRIPT_FILENAME'] ) ) {
+    die ('<h1>Nice try, Charlie!</h1><h2>Try going to the form page here: </h2><br> <div class=\"buttonContainer\"> <a href="formPage.php"><button type=\"button\">GO BACK TO FORM</button></a></div>');
+}
+
+require_once 'functions.php';
+
+$connection = dbConn();
```

Comment:
Did you not need the DB connection at the top of the page as a PDO doc?

Reply:
Just got the all-clear on that.